### PR TITLE
Created Contributing.md file

### PR DESCRIPTION
**Pull request by pbelokon**
Closes https://github.com/Would-You-Bot/website/issues/48.

The PR adds Contributing.md file with contribution guidelines.

Please let me know in case of any feedback.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contribution Guidelines
+
+We are all eager to work on official website for the Would You Discord bot, and this document will help you to get started!
```
Comment (mezotv) [inline Contributing.md:3]: This line sounds a bit weird. Would be nice to change it to something more human like. deepl.com/write can help!

**Review by mezotv — CHANGES_REQUESTED** [review]

File: Contributing.md (hunk 1 above)
Comment (pbelokon) [reply Contributing.md:3]: sure will do so

Comment (pbelokon) [reply Contributing.md:3]: @mezotv  I have something in mind how does this sound: 

Thank you for showing an interest in contributing! All kinds of contributions are valuable to us. In this guide, we will help you to get started!

Comment (mezotv) [reply Contributing.md:3]: Maybe like this "Thank you for your interest in contributing! All types of contributions are valuable to us. This guide will help you get started!"

Comment (pbelokon) [reply Contributing.md]: Changed with the new commit

**Comment by mezotv** [comment]
I will take a look tomorrow. Thanks!

**Review by Podskio — APPROVED** [review]

**Review by mezotv — APPROVED** [review]
Looks good!

**Pull request merged**